packages/reports/app/styles/navi-reports/components/grouped-list.less

```diff
@@ -60,49 +60,49 @@
       &-container {
         position: relative;
       }
+    }
 
-      &-label {
-        &:before {
-          border-radius: 1px;
-          box-shadow: 0 0 0 2px rgba(54, 151, 242, 0.5);
-          content: '';
-          display: inline-block;
-          height: 8px;
-          margin: 0 5px;
-          transition: all 0.2s;
-          width: 8px;
-        }
+    &-checkbox + &-label {
+      &:before {
+        border-radius: 1px;
+        box-shadow: 0 0 0 2px rgba(54, 151, 242, 0.5);
+        content: '';
+        display: inline-block;
+        height: 8px;
+        margin: 0 5px;
+        transition: all 0.2s;
+        width: 8px;
       }
+    }
 
-      &:hover + &-label:before {
-        box-shadow: 0 0 0 2px @denali-brand-600;
-      }
+    &-checkbox:hover + &-label:before {
+      box-shadow: 0 0 0 2px @denali-brand-600;
+    }
 
-      &:focus + &-label:before {
-        box-shadow: 0 0 0 2px rgba(54, 151, 242, 0.5);
-      }
+    &-checkbox:focus + &-label:before {
+      box-shadow: 0 0 0 2px rgba(54, 151, 242, 0.5);
+    }
 
-      &:checked + &-label:before {
-        box-shadow: 0 0 0 2px @denali-brand-600;
-        background: @denali-brand-600;
-      }
+    &-checkbox:checked + &-label:before {
+      box-shadow: 0 0 0 2px @denali-brand-600;
+      background: @denali-brand-600;
+    }
 
-      &:checked + &-label:after {
-        border: solid #fff;
-        border-width: 0 2px 2px 0; // creates the inverted "L" shape
-        box-sizing: border-box;
-        content: ' ';
-        display: block;
-        height: 8px;
-        left: 6px;
-        position: absolute;
-        top: 4px;
-        transform: rotate(45deg);
-        width: 5px;
-      }
+    &-checkbox:checked + &-label:after {
+      border: solid #fff;
+      border-width: 0 2px 2px 0; // creates the inverted "L" shape
+      box-sizing: border-box;
+      content: ' ';
+      display: block;
+      height: 8px;
+      left: 6px;
+      position: absolute;
+      top: 4px;
+      transform: rotate(45deg);
+      width: 5px;
     }
 
-    &-label {
+    span&-label {
```

Comment:
Space?
```suggestion
    span &-label {
```

Reply:
No, this is supposed to be `span.grouped-list__item-label` as opposed to `label.grouped-list__item-label`

Reply:
Ah gotcha just looked off to me